aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -253,43 +250,64 @@ impl VersionedValue {
 
         // Finally, resolve if needed with the base value.
         self.base_value
+            .clone()
```

Comment:
we can clone inside and_then so we do not clone snapshots and string bytes?

Reply:
those should never happen, but sure :)